run.py

```diff
@@ -2,7 +2,7 @@
 import asyncio
 import time
 import argparse
-from major_tom.major_tom import MajorTom
+from mt_gateway_api.gateway_api import GatewayAPI
```

Comment:
It's odd that the package is called `major-tom-gateway-api` in requirements.txt and `mt_gateway_api` when importing. Can these be the same?

Reply:
Yeah it was weird. Pypi's tutorial sets it up like that, and it's weird. I'd have to change the packages